### DISCUSSION TITLE
Linux Automated Setup Guide: Step 4 error

### DISCUSSION THREAD

**Discussion by goawaydude** [hlky/stable-diffusion-webui#849] [Q&A]
I previously had everything functional after working around parts of the original retard guide. Now I have installed Anaconda in prep for following the automated guide. I receive the following error on step 4 after typing "./linux-sd.sh":

![Untitled](https://user-images.githubusercontent.com/58157838/189193373-6a1ad2e9-4ae0-4247-9d39-2c3bb6ffc087.png)

Am I doing something wrong?

**Comment by KyuubiYoru** [comment]
Only thing i needed to do on Linux was, navigate to stable-diffusion-webui and run:
`conda env create -f environment.yaml`
`conda activate ldm`
`python3 scripts/relauncher.py`

**Comment by cstueckrath** [comment]
you did not download the script, but the webpage with the script.
Follow the link, click on "Raw", use that link
